places/vicuk-london.md

```diff
@@ -0,0 +1,9 @@
+## London
+
+### Fun Facts
```

Comment:
@jibralta Big Ben is the bell, not the clock tower. Its chime is in the key of E.